controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -84,6 +93,14 @@ private StreamMetadataTasks(final StreamMetadataStore streamMetadataStore,
         this.setReady();
     }
 
+    public Void initializeStreamWriters(final ClientFactory clientFactory,
+                                        final String streamName) {
+        this.requestStreamName = streamName;
```

Comment:
I gather from the Void return type this is going to be used in a completable future, which means it may be accessed via multiple threads. So these member variables need to be guarded by a lock. (Or you could have a factory do this, and then they would be final)